src/lib/select/select.ts

```diff
@@ -890,31 +893,29 @@ export class MdSelect extends _MdSelectMixinBase implements AfterContentInit, On
 
   /** Calculates the scroll position and x- and y-offsets of the overlay panel. */
   private _calculateOverlayPosition(): void {
+    this._triggerHeight = this.trigger.nativeElement.getBoundingClientRect().height;
+    this._triggerFontSize = this._measureFontSizeEl.nativeElement.getBoundingClientRect().height;
```

Comment:
Do you know if `getComputedStyle` is guaranteed to give me a `px` value for `font-size` (as opposed to `em`, `%`, etc)? That was my main motivation for not doing`getComputedStyle`. Also, I assume `getComputedStyle` is no worse from a performance perspective?